app/config/parameters_common.yml

```diff
@@ -28,3 +28,4 @@ parameters:
         Shopsys\FrameworkBundle\Model\Transport\Transport: Shopsys\ShopBundle\Model\Transport\Transport
         Shopsys\FrameworkBundle\Model\Customer\BillingAddress: Shopsys\ShopBundle\Model\Customer\BillingAddress
         Shopsys\FrameworkBundle\Model\Customer\User: Shopsys\ShopBundle\Model\Customer\User
+        Shopsys\FrameworkBundle\Model\Administrator\Administrator: Shopsys\ShopBundle\Model\Administrator\Administrator
```

Comment:
what about \Tests\ShopBundle\Functional\EntityExtension\EntityExtensionTest? 
Currently, this entity is not used in tests and it could be surprising in the future

Reply:
That tests uses extension map and then adds some entities that it uses.

Reply:
Okay, that's valid, thanks. 

But then the tests should be fixed, don't they? I'm talking for example about `AdministratorRepositoryTest`, where 
`$administrator` is annotated as a Framework entity.
I know that in this test it doesn't matter (no extra fields are used), but couldn't it be confusing in the future?

Reply:
Well we do not do that. @vitek-rostislav is just implementing fixer for updating annotations, so maybe he will solve this.

Reply:
Okay, I don't agree with "we don't do that", but fixer in progress is a strong enough argument to keep me in peace 😄